frame/bags-list/remote-tests/Cargo.toml

```diff
@@ -15,6 +15,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 # frame
 pallet-staking = { path = "../../staking", version = "4.0.0-dev" }
+pallet-stake-tracker = { path = "../../stake-tracker", version = "4.0.0-dev" }
```

Comment:
```suggestion
pallet-stake-tracker = { path = "../../stake-tracker", version = "1.0.0-dev" }
```

Reply:
@kianenigma this version is consistent with all the other pallet versions, I think there is no sense to do 1.0.0-dev, if all the others are 4.0.0-dev.